app/script/conversation/ConversationRepository.js

```diff
@@ -2852,6 +2852,9 @@ z.conversation.ConversationRepository = class ConversationRepository {
         }
       }
 
+      if (conversationEntity.is_cleared()) {
+        conversationEntity.cleared_timestamp(0);
+      }
```

Comment:
I think this should only happen if we did end up mapping a `messageEntity` onto the conversation.